plugins/linux/ifplugin/interface_config.go

```diff
@@ -992,3 +969,60 @@ func addressExists(configured []netlink.Addr, provided *net.IPNet) bool {
 	}
 	return false
 }
+
+// ResolveCreatedInterface resolves a new vpp interfaces
+func (plugin *LinuxInterfaceConfigurator) ResolveCreatedVPPInterface(ifConfigMetaData *vppIf.Interfaces_Interface) error {
+	plugin.log.Infof("Linux IF configurator: resolve created vpp interface %v", ifConfigMetaData)
+
+	if ifConfigMetaData.Type != vppIf.InterfaceType_TAP_INTERFACE {
+		return nil
+	}
+
+	var hostIfName string
+	if ifConfigMetaData.Tap != nil && ifConfigMetaData.Tap.HostIfName != "" {
+		hostIfName = ifConfigMetaData.Tap.HostIfName
+	} else if ifConfigMetaData != nil && ifConfigMetaData.Name != "" {
+		hostIfName = ifConfigMetaData.Name
+	} else {
+		return nil
+	}
+	return plugin.configureTapInterface(hostIfName, nil)
+}
+
+// ResolveDeletedInterface resolves removed vpp interfaces
+func (plugin *LinuxInterfaceConfigurator) ResolveDeletedVPPInterface(ifConfigMetaData *vppIf.Interfaces_Interface) error {
+	plugin.log.Infof("Linux IF configurator: resolve deleted vpp interface %v", ifConfigMetaData)
+
+	if ifConfigMetaData.Type != vppIf.InterfaceType_TAP_INTERFACE {
+		return nil
+	}
+
+	var hostIfName string
+	if ifConfigMetaData.Tap != nil && ifConfigMetaData.Tap.HostIfName != "" {
+		hostIfName = ifConfigMetaData.Tap.HostIfName
+	} else if ifConfigMetaData != nil && ifConfigMetaData.Name != "" {
```

Comment:
This check is pointless because if the `ifConfigMetaData` will be `nil`, the program will crash on line `996` due accessing its fields.

Reply:
:-) you are right. I will change it